apps/src/music/redux/musicRedux.ts

```diff
@@ -55,6 +55,13 @@ export interface MusicState {
     canUndo: boolean;
     canRedo: boolean;
   };
+  /** A callout that's being shown.  The index lets the same callout be
+   * reshown multiple times in a row.
+   */
+  showCallout: {
```

Comment:
Similarly, should this be part of lab2 redux if we want to enable this for all lab2 labs?